Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -534,9 +534,17 @@ internal async Task<string> SelectChild(TreeNode treeNode)
                 {
                     return cs.Child;
                 }
-                if ((bool)await this.EvaluateDynamicProperty(cs.ShouldSelect, typeof(bool)).ConfigureAwait(false))
+                var evaluationResult = await EvaluateDynamicProperty(cs.ShouldSelect, typeof(bool)).ConfigureAwait(false);
+                try
```

Comment:
Your branch name should have the name: feature/foo
That way, the CI/CD pipelines will automatically pick up your branch and PR. Please try sending a new PR with correct branch name format, ty!